### DISCUSSION TITLE
Vue upgrades

### DISCUSSION THREAD

**Discussion by CrawfordW** [quasarframework/quasar#11419] [General - Components / Directives / etc] [answered]
When I upgrade Quasar version does it also get the latest Vue version, or do I have to manage that myself?
Currently running Quasar v 2.3.2 with app v 3.2.2

devTools keeps giving me a bunch of warning like this:
[Vue warn]: injected property "something" is a ref and will be auto-unwrapped and no longer needs `.value` in the next minor release

It would be nice to get rid of these warnings by getting this 'next minor release' and making the needed changes!

**Comment by MilosPaunovic** [comment, marked as answer]
`quasar/app` keeps track of the `Vue` releases, so, you just have to keep your `quasar/app` up-to-date.

**Reply by CrawfordW** [reply]
Good. Thanks.
